cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -456,4 +477,29 @@ protected void setRackConfig(RackConfig rackConfig) {
     protected void setInitImage(String initImage) {
         this.initImage = initImage;
     }
+
+    @Override
+    protected Map getDefaultLogConfig() {
```

Comment:
Why shouldn't this be a `Properties` instead of a `Map`?

Reply:
Cannot we use a resource file for that instead of hard coding this default settings?